validphys2/src/validphys/coredata.py

```diff
@@ -51,6 +51,28 @@ class FKTableData:
     sigma: pd.DataFrame
     metadata: dict = dataclasses.field(default_factory=dict, repr=False)
 
+    def __post_init__(self):
+        self._fktable = None
+
+    @property
+    def protected(self):
```

Comment:
I'm having trouble understanding this and why we need it. What is a repetition flag?

Reply:
Do I understand correctly that the only place where this ever comes up is ratios and everything else could operate transparently?

Reply:
> In addition in the original grids some of these are stored as grids of n points where everything is 0 but the first.

Which cases? Wouldn't that break the existing convolution code which does nothing to handle that?

Reply:
as @scarlehoff said this is done in apfelcomb, so before convolutions appear

Reply:
I wonder if some explicit concept of normalization grid that doesn't come with a datapoint index could be made to work. With the rough semantics that it is only one cross section and operations broadcast like numpy would. Implementation wise it would be stored like a normal fktable (and could act as one if we wanted to e.g. fit the total cross setion) but would drop the data index when loaded as a normalization.

Reply:
...so e.g. `with_cuts` would be a no op.

Reply:
Yes, I guess this is what I'm achieving with the `protected` property. The table itself can be loaded if there is no repetition flag in the yaml file.

I guess this can be made a fktable-only property by moving the cfactor application to the FKTableData object?

Reply:
I suppose moving the cfactors application might make it neater. One reason why it wasn't done was that I didn't want to have FKTableData objects in intermediate states lying around and instead there is one way to load once and for all.

That said I was rather thinking in having a slightly different object with the requisite behaviour rather than patching up FKTableData to support both.